src/llama_recipes/utils/__init__.py

```diff
@@ -3,5 +3,5 @@
 
 from llama_recipes.utils.memory_utils import MemoryTrace
 from llama_recipes.utils.dataset_utils import *
-from llama_recipes.utils.fsdp_utils import fsdp_auto_wrap_policy
+from llama_recipes.utils.fsdp_utils import fsdp_auto_wrap_policy, hdsp_device_mesh
```

Comment:
Should `hdsp_device_mesh` be changed to `hsdp_device_mesh`?
```suggestion
from llama_recipes.utils.fsdp_utils import fsdp_auto_wrap_policy, hdsp_device_mesh
```